main/aperture.ts

```diff
@@ -22,6 +22,9 @@ let apertureOptions: ApertureOptions;
 let recordingName: string | undefined;
 let past: number | undefined;
 
+export let overallDuration = 0;
+export let currentDurationStart = 0;
```

Comment:
Woops, forgot to lint. Also I didn't know mutable exports were bad, what's another method I can use to store the current duration? Or maybe there's a better way to get the length of the current recording that I don't know of? I haven't really used ffmpeg very much.

Reply:
It's not clear to me why that's bad. I think you can just use an ESLint ignore comment here.

Reply:
I think it would be useful to expose this information in https://github.com/wulkano/aperture-node, but that's not required by this PR. But maybe you can add a TODO comment about it.

Reply:
Rather than a TODO comment, I was thinking of writing an issue in [aperture-node](https://github.com/wulkano/aperture-node) for exposing the duration. Does that sound like a good idea?

Reply:
> I was thinking of writing an issue in [aperture-node](https://github.com/wulkano/aperture-node) for exposing the duration. Does that sound like a good idea?

:+1: But still write a TODO that links to that issue.

Reply:
Done! 😄